src/main-app.component.ts

```diff
@@ -1,14 +1,16 @@
-import { Component } from '@angular/core';
+import {Component, ViewEncapsulation} from '@angular/core';
 import { ROUTER_DIRECTIVES } from '@angular/router';
 
 import { SideMenuComponent } from './components/sidemenu/side-menu.component';
+import { NotificationComponent } from './components/notification/notification.component';
 
 @Component({
     moduleId: module.id,
     selector: 'main-app',
     template: require('./main-app.component.html'),
     styles: [require('./main-app.component.scss')],
-    directives: [SideMenuComponent, ROUTER_DIRECTIVES]
+    directives: [SideMenuComponent, ROUTER_DIRECTIVES, NotificationComponent],
```

Comment:
Please move the NotificationComponent to AppModule declarations. Over the time all the directives should be there since rc.5.